### DISCUSSION TITLE
Scene select based on "perfomer is in the scene with higher rankng than x stars" possible?

### DISCUSSION THREAD

**Discussion by becks0815** [stashapp/stash#4652] [General]
Is there a way to select only scenes including/excluding the ones where a performer is in the scene with a given ranking (5 stars, higher than x stars, lower than x/stars) and use this filter in combination with others, e.g. by selecting a specific studio?

If not, this could be an improvement idea for the future.

An alternative short term solution would be if someone knows the SQL statement to add a tag to all scenes with above criteria (e.g. add a tag to all scenes with a performer ranked 5 stars).

**Comment by feederbox826** [comment]
https://github.com/stashapp/stash/issues/6357
